.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should not be here. Please remove it and maybe add `.idea` to our gitignore

Reply:
Done. Thank you very much for the review.